lib/program-helper.js

```diff
@@ -59,13 +59,17 @@ function fileExists(cmdPath) {
         return false;
     }
 }
-
+/**
+ * @typedef {Object} HelperType
+ * @prop {() => string} getTSCCommand
+ * @prop {() => string[]} resolveTSFiles
+ * @prop {() => Command} getOptions
+ */
+/** @type {HelperType} */
```

Comment:
Correct but I will likely move it when I do the cleanup that I proposed in PR #9.